math/prime_numbers.cpp

```diff
@@ -3,23 +3,23 @@
 
 std::vector<int> primes(int max) {
     max++;
-    std::vector<int> res;
```

Comment:
Why not use a _real_ variable name?  Does `res` stand for reservation, resort, residue.  The program is just as fast and _much_ more maintainable if we use longer, more intuitive variable names.